src/Panel/Controller/Controller.php

```diff
@@ -30,6 +31,22 @@ public function __construct()
 		$this->site    = $this->kirby->site();
 	}
 
+	protected function i18n(
+		string|array|null $key,
+		array|null $data = null
+	): string|null {
+
+		if ($key === null) {
+			return null;
+		}
+
+		if ($data === null) {
+			return I18n::translate($key, $key);
+		}
+
+		return I18n::template($key, $data);
+	}
+
```

Comment:
We keep implementing the same method in various places. Components, Fields and now the Controllers. I wonder if we should add a I18n trait to the Toolkit for this